### ISSUE TITLE
Cryptic error given when running hippunfold before `hippunfold_download_models` is run

### ISSUE THREAD

**Issue by akhanf** [khanlab/hippunfold#193]
The error message is cryptic because of the generic `copy_to_results` rule, since any missing file for a rule ends up then checking that rule, where is ultimately fails.. 

I'd like to actually get rid of this generic rule, will make updating the workflow much easier.. was a lazy addition in the first place.  I will also add some explicit checks on whether the trained models are downloaded already, and provide a useful warning if any are missing.

**Comment by akhanf** [comment]
By the way, this is the error message:
```
WorkflowError:
MissingInputException: Missing input files for rule copy_to_results:
    output: hippunfold/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfaces.spec
    wildcards: file=sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfaces.spec
    affected files:
        work/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfaces.spec
WorkflowError:
    MissingInputException: Missing input files for rule copy_to_results:
        output: hippunfold/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_gyrification.shape.gii
        wildcards: file=sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_gyrification.shape.gii
        affected files:
            work/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_gyrification.shape.gii
    WorkflowError:
        MissingInputException: Missing input files for rule copy_to_results:
            output: hippunfold/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfarea.shape.gii
            wildcards: file=sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfarea.shape.gii
            affected files:
                work/sub-001/surf/sub-001_hemi-L_space-T1w_den-0p5mm_label-hipp_surfarea.shape.gii
        CyclicGraphException: Cyclic dependency on rule warp_gii_to_native.
```

**Comment by akhanf** [comment]
working on this in #194

**Comment by akhanf** [comment]
Closed with #194
